src/App/Osu/OsuOptionsMenu.cpp

```diff
@@ -1019,6 +1019,7 @@ OsuOptionsMenu::OsuOptionsMenu(Osu *osu) : OsuScreenBackable(osu)
 	addCheckbox("SuddenDeath restart on miss", "Skips the failing animation, and instantly restarts like SS/PF.", convar->getConVarByName("osu_mod_suddendeath_restart"));
 	addCheckbox("Show Skip Button during Intro", "Skip intro to first hitobject.", convar->getConVarByName("osu_skip_intro_enabled"));
 	addCheckbox("Show Skip Button during Breaks", "Skip breaks in the middle of beatmaps.", convar->getConVarByName("osu_skip_breaks_enabled"));
+	addCheckbox("Auto restart","Skips the failing animation, and instantly restarts", convar->getConVarByName("osu_auto_reset"));
```

Comment:
1. Move the checkbox to directly below the "SuddenDeath restart on miss".
2. Rename it from "Auto restart" to "Restart on death".
3. Add a sentence ending dot `.` at the end of the tooltip string, to match all the other tooltip strings' styles.
4. Fix the coding style by adding a space after the first comma `,` in order to make reading the code easier by not having everything squished together.

You might now still think that "Why are these small style changes even relevant, who cares?" but having a consistent style enforced is **very important** for larger projects.